docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,942 @@
+</SdkApi>
+
+<SdkApi
+  name="addEventProcessor"
+  signature="function addEventProcessor(processor: EventProcessor): void"
+  parameters={[
+    {
+      name: "processor",
+      type: "(event: Event, hint: EventHint) => Event | null | Promise<Event | null>",
+    },
+  ]}
+>
+  Adds an event processor to the SDK. An event processor receives every event
+  before it is sent to Sentry. It can either mutate the event (and return it) or
+  return `null` to discard the event. Event processors can also return a
+  promise, but it is recommended to use this only when necessary as it slows
+  down event processing.
+</SdkApi>
+
+<SdkApi
+  name="addIntegration"
+  signature="function addIntegration(integration: Integration): void"
+>
+  Adds an integration to the SDK. This can be used to conditionally add
+  integrations after `Sentry.init()` has been called. Note that it is
+  recommended to pass integrations to `init` instead of calling this method,
+  where possible.
+</SdkApi>
+
+<SdkApi name="lazyLoadIntegration" signature="function lazyLoadIntegration(name: string, scriptNonce?: string): Promise<Integration>" categorySupported={['browser']}>
+  Lazy load an integration. This expects the name to be e.g. `replayIntegration`. It will load the script from the CDN, and return a promise that resolves to the integration, which can then be added to the SDK using `addIntegration`:
+
+```javascript
+Sentry.lazyLoadIntegration("replayIntegration")
+  .then((integration) => {
+    Sentry.addIntegration(integration);
+  })
+  .catch((error) => {
+    // Make sure to handle errors here!
+    // This rejects e.g. if the CDN bundle cannot be loaded
+  });
+```
+
+If you use a bundler, using e.g. `const { replayIntegration } = await import('@sentry/browser')` is recommended instead.
+
+</SdkApi>
+
+## Scopes
+
+<SdkApi name="getCurrentScope" signature="function getCurrentScope(): Scope">
+  Returns the current scope. See{" "}
+  <PlatformLink to="/enriching-events/scopes/">Scopes</PlatformLink> for more
+  information.
+
+Note that in most cases you should not use this API, but instead use `withScope` to generate and access a local scope. There are no guarantees about the consistency of `getCurrentScope` across different parts of your application, as scope forking may happen under the hood at various points.
+
+</SdkApi>
+
+<SdkApi
+  name="withScope"
+  signature="function withScope(callback: (scope: Scope) => void): void"
+>
+  Forks the current scope and calls the callback with the forked scope. See{" "}
+  <PlatformLink to="/enriching-events/scopes/">Scopes</PlatformLink> for more
+  information.
+</SdkApi>
+
+<SdkApi
+  name="getIsolationScope"
+  signature="function getIsolationScope(): Scope"
+>
+  Returns the current isolation scope.
+</SdkApi>
+
+<SdkApi
+  name="withIsolationScope"
+  signature="function withIsolationScope(callback: (scope: Scope) => void): void"
+>
+  Forks the current isolation scope and calls the callback with the forked
```

Comment:
super-l: Not sure if "forking" is as widely understood as "cloning" but happy to leave this up to you

```suggestion
  Clones the current isolation scope and calls the callback with the cloned
```

Reply:
sounds good!